src/api/theme.h

```diff
@@ -24,11 +24,11 @@ enum web_theme {
 } __attribute__ ((packed));
 
 struct web_themes{
-	const enum web_theme id;
 	const char *name;
 	const char *description;
-	const bool dark;
 	const char *color;
+	const enum web_theme id;
+	const bool dark;
```

Comment:
Rearranging `struct` content so save space

Reply:
Maybe of interest here: The "old" way takes 40 bytes of total size on 64 bit (20 bytes on 32 bit). Thew new ordering is 32 bytes on 64 bit, unchanged 20 bytes on 32 bit systems.

Reply:
Thanks!